docs/tutorials/projects/helmet.md

```diff
@@ -64,7 +64,7 @@ Get your hardware ready and connected to the Viam platform:
 
 2. Make sure your computer (whether it's a personal computer or an SBC) is connected to adequate power, and turn it on.
 
-3. If you haven't already, [create a new machine](/fleet/machines/#add-a-new-machine) in the [Viam app](https://app.viam.com) and follow the instructions to [install `viam-server`](/get-started/installation/) on the computer you're using for this project.
+3. If you haven't already, [create a new machine](/fleet/machines/#add-a-new-machine) in the [Viam app](https://app.viam.com) and follow the instructions to [install `viam-server`](/get-started/installation/), the software you will use to operate your machine, on the computer you're using for this project.
```

Comment:
```suggestion
3. If you haven't already, [add a new machine](/fleet/machines/#add-a-new-machine) in the [Viam app](https://app.viam.com) and follow the setup instructions to install `viam-server` on the computer you're using for this project and connect to the Viam app.
```
A bit simpler - I'll assign myself the ticket to go through all tutorials to change it to something like this at the end